Simplified.xcodeproj/project.pbxproj

```diff
@@ -2815,15 +2813,13 @@
 				E65977C51F82AC91003CD6BC /* NYPL_Launch_Screen.storyboard in Resources */,
 				03F94CCF1DD627AA00CE8F4F /* Accounts.json in Resources */,
 				A823D81B192BABA400B55DE2 /* InfoPlist.strings in Resources */,
-				219901F424FD2DE9001BC727 /* jwk.json in Resources */,
```

Comment:
this was added to the app build, but it's only used by unit tests as far as I know.